scripts/infinity_grid.py

```diff
@@ -347,11 +348,11 @@ def show(self, is_img2img):
         return True
 
     def ui(self, is_img2img):
+        try_init()  # Moved to the beginning to ensure modes are registered before UI is built
```

Comment:
(A) it was already at the beginning, the move doesn't really make sense
(B) shouldn't leave change comments in code, make the change in code and leave notes on git